build.gradle

```diff
@@ -105,18 +106,33 @@ configure(javaProjects) {
             dependency("org.opensaml:opensaml:2.6.4")
             dependency("org.python:jython-standalone:2.7.1")
             dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
+            dependencySet(group: "com.google.protobuf", version: "3.4.0") {
```

Comment:
(OCD) Can you put these in alphabetical order with all the rest?

Reply:
Also it seems there's already a version of `protobuf` managed by the [Spring platform](https://docs.spring.io/platform/docs/Brussels-SR5/reference/htmlsingle/#appendix-dependency-versions) should we just go with that or is there a reason you bumped it to 3.4.0. I see that version is 2.6.1. There is a standard way to override Spring platform versions and that's to put a property probably something like `protobuf.version` to override it.

Reply:
The upcoming Spring 5 and Boot 2 supports proto 3.x, and recommends 3.4.0. https://github.com/spring-io/platform/blob/d5f74380022d67d03b02feebf30963bc52913f40/platform-bom/pom.xml#L136

If you're a ways out from releasing this, it may be worth coding to the Boot 2.x milestone releases.

Reply:
Agreed that we will be targeting boot 2.x. We're going to have to follow the upgrade steps though not part of this: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.0-Release-Notes